Azure/azure-cli
# [AppService] BREAKING CHANGE: Fix regression where you can't use a runtime version with a space in the name

**Description**
Fixes issue #16066 

Before, az webapp up/create allowed you to specify `--runtime` parameter with delimiters "|", " ", and ":".
This parsing caused runtimes with a space (" ") in it to not be parsed correctly.

In this PR, removing ability to use space (" ") as a delimiter so that runtimes with a space in it get parsed correctly again

**Testing Guide**
`az webapp create -n {} -g {} --runtime "'java|1.8|Java SE|8" --os-type Windows` should work


---

This checklist is used to make sure that common guidelines for a pull request are followed.
